### DISCUSSION TITLE
Viewport feature in Custom Editor UI

### DISCUSSION THREAD

**Discussion by skpulipaka26** [puckeditor/puck#435] [General]
Is there a way to get viewport icons and functionality with custom editor ui. I dived into the src and noticed there's a Canvas element which has this functionality. Are there any plans to expose this component? If not, any workaround?

**Comment by skpulipaka26** [comment]
Found my answer in discord chat. Thank you.

**Reply by brunowego** [reply]
@skpulipaka26 can you share here the answer for your question? Thanks.
